docs/rfcs/1013-parsec-account.md

```diff
@@ -26,485 +36,675 @@ That is how we came to the idea of storing the device on a remote server.
 - That will solve the issue of the browser cleaning its data.
 - The devices would be accessible from anywhere (given the user is online).
 
-## Goals
+Hence this chain of RFCs (1013, 1014, 1015, 1016, 1017, 1018 and 1019) to specify
+all those new features.
+
+### 2.2 - Relationship with existing invitation & enrollment
+
+Currently, the enrollment process starts by providing an invitation URL to the claimer.
+
+Once this URL opened with the Parsec client, enrollment dance can proceed right away.
+
+With this RFC, the enrollment process must occur within the context of a Parsec Account:
+
+- If the user is connected to Parsec Account, he will see the list of pending invitations.
+- If the user is not connected to Parsec Account and opens an invitation URL, the Parsec
+  client will ask him to connect to Parsec Account with the email corresponding to the
+  invitation (obtained by the querying the server with the `invite_info` command).
+- If the user is connected to Parsec Account and opens an invitation URL:
+  - If the invitation's email correspond to the Parsec Account, the enrollment can start
+    right away.
+  - Otherwise the Parsec client will ask the user to connect to the Parsec Account
+    matching the invitation (while also prompting the possibility to create such
+    Parsec Account).
+
+### 2.3 - Relationship with existing device keys files
+
+Currently, once enrollment is done, a device keys files is written to the machine
+hard disk. As its name suggest, this file contains, among other things, the device keys.
+
+There is no change needed to this behavior.
+
+Indeed: once logged in to Parsec Account, the user should see the list of identities
+(i.e. couple organization ID + user ID) he can connect to.
+
+Typically identities can be in the following states:
+
+1. Organization expired
+2. User revoked
+3. Valid, no device locally, and no registration device
+4. Valid, no device locally and registration device available
+5. Valid, and device present locally
+
+The Parsec Account API allows to obtain:
+
+- The full list of identities
+- Which organization is expired
+- Which user is revoked
+- Which organization allows registration devices
+- The list of registration devices
+
+From there, listing the available devices locally allows to fully determine which
+identity can connected to.
 
-- Describe how to store/access the user devices on a remote service.
-- Propose an API to implement this feature.
-- Focus on allowing users to connect using a password
+> Note: In case of offline login to Parsec Account, we only use the login as a filter
```

Comment:
It's the first (and only ?) mention of offline login, I'd add a note to define it.